tests/integration/features/chat/read-status.feature

```diff
@@ -0,0 +1,35 @@
+Feature: public
+  Background:
+    Given user "participant1" exists
+    Given user "participant2" exists
+
+  Scenario: User toggles the read privacy
```

Comment:
would be nice to also add a scenario with multiple users with "public" setting and demonstrate the "last common read message id", which would validate the "min()" function.

or is this already somehow covered in the second part of this scenario ?
a separate scenario could make it clearer that it is about the "common read", not directly about the "privacy flag"

Reply:
yes it is, as it always considers your and the other persons read status. let me try to make it explicit